### ISSUE TITLE
Disable /Gm (minimal rebuild) for newly created projects.

### ISSUE THREAD

**Issue by paulhoux** [cinder/Cinder#1158]
The /Gm switch disables multi-processor building. In practice, building on all available processors reduces compile times more than any minimal rebuild could.

**Comment by richardeakin** [comment]
It would be nice to see numbers around this, especially including when you modify a file and recompile. I've found that in that case, the minimal rebuild is faster and ideal for user app projects. But I haven't done any close measurements with VS 2015 and alot has changed around compile times in that version.

**Comment by paulhoux** [comment]
OK, did some profiling. I cleanly rebuilt Cinder Release Win32 several times, using the build timer:

Minimal Rebuild, No Multiprocessor | No Minimal Rebuild, Multiprocessor
--------------------|------------------
200.095 sec|67.376 sec
202.173 sec|65.190 sec
201.552 sec|66.915 sec

Single file (AppBase.cpp) forcing it to rebuild be adding a space

Minimal Rebuild, No Multiprocessor | No Minimal Rebuild, No Multiprocessor | No Minimal Rebuild, Multiprocessor
--------------------|------------------|-------
3.692 sec|3.582 sec|3.306 sec
3.524 sec|3.273 sec|3.329 sec
3.511 sec|3.326 sec|3.294 sec

If these tests are representative, it is clear that enabling multiprocessor compilation outweighs minimal rebuild, especially when compiling more than just a few files.

Tested on Intel Core i7-4720HQ @ 2.60GHz (4 cores, hyperthreading)

**Comment by richardeakin** [comment]
Hm, looks like `/Gm` isn't giving us anything over `/MP`. Though your numbers are for rebuilding libcinder, so I tried with the FallingGears sample to see what they look like for a user app (which point at a similar conclusion). Tested on an Intel i7-4790 4gz with ssd drive, VS2015 running v120 toolset:

```
FallingGears sample

/Gm rebuild:
16.9s
20.3s
20.0s

/Gm build /w added line to SceneController.cpp
3.76s
4.3
4.2

/MP rebuild:
8.8s
8.7s
8.5s

/MP build /w added line to SceneController.cpp
4.0
3.5
3.45
```

A full rebuild is nearly twice as fast 

For libcinder, I am seeing that neither `/MP` nor `/Gm` are set. We should probably add `/MP` for all targets.

For samples, I am seeing that many (most) have both `/MP` and `/Gm` set, and you get a warning that the `/MP` setting will have no effect because of this. We should probably remove the `/Gm`.
